.golangci.yml

```diff
@@ -565,6 +565,483 @@ issues:
     - path: comp/dogstatsd/packets/packet_manager_windows.go
       linters:
         - pkgconfigusage
+    - path: comp/agent/autoexit/autoexitimpl
+      linters:
+        - compimpl
```

Comment:
I think this is fair point and I am wondering if we really want to enforce it (even during a regular review).

Reply:
A significant portion of these exceptions are using the [classic component style](https://github.com/DataDog/datadog-agent/blob/baa23e09cd4e211b6f86c60479c8e2c4370d5435/tasks/components.py#L81). Is it necessary (according to the yaml style formatter) to list all these alphabetically? Otherwise, could we separate this list into classic components, and new components that don't pass the linter. That way we could at least have an easier time fixing up the new style components.